src/utils.ts

```diff
@@ -109,15 +109,50 @@ export const getUniqueItemsInArray = (array: any) => {
 
 export const getNorwegianLabelFirst = (listOfLabels: MultipleLabelType[]): string => {
 	let label: string = EMPTY;
-	if (listOfLabels.length > 1) {
-		listOfLabels.forEach((element: MultipleLabelType) => {
-			if (element.language === 'nb') {
-				label = element.value;
-				return;
-			}
-		});
+	const firstItemInArray = listOfLabels[0].value;
+
+	if (listOfLabels.length === 1) {
+		label = firstItemInArray;
 	} else {
-		label = listOfLabels[0].value;
+		label =
+			getNorwegianBokmalLabel(listOfLabels) ||
+			getNorwegianNynorskLabel(listOfLabels) ||
+			getEnglishLabel(listOfLabels) ||
+			firstItemInArray;
 	}
 	return label;
 };
+
+export const getNorwegianBokmalLabel = (list: MultipleLabelType[]): string | undefined => {
+	let label: string = EMPTY;
+	list.forEach(element => {

```

Comment:
Is there a list::filter method that can be used?

Reply:
yes, but we would need to chain the filter and map functions, which is much less performant than a foreach.
source: https://gomakethings.com/how-performant-are-modern-array-methods-vs-old-school-for-loops-in-vanilla-js/